addons/account/tests/test_chart_template.py

```diff
@@ -184,3 +184,14 @@ def test_update_taxes_removed_from_templates(self):
 
         # if only the fiscal position mapping has been removed, it won't be recreated
         self.assertEqual(len(fiscal_position.tax_ids), 0)
+
+    def test_update_taxes_conflict_name(self):
+        chart_template_xml_id = self.chart_template.get_external_id()[self.chart_template.id]
+        template_vals = self.tax_1_template._get_tax_vals_complete(self.company_1)
+        template_vals['amount'] = 20
```

Comment:
Why ?

Reply:
We need the template to be a bit different from the initial one to trigger re-creation/duplication with name conflict, so I changed the amount to get to that state.